### PR TITLE
-Fixed DirtyDrivin windowed mode

### PR DESCRIPTION
**Pull request by 00C0FFEE**

### CODE REVIEW

File: OpenParrot/src/Functions/Games/Other/DirtyDrivin.cpp [hunk 1]
```diff
@@ -102,27 +143,45 @@ DWORD WINAPI InputRT9(LPVOID lpParam)
...
 		Sleep(deltaTimer);
 	}
 
 	return 0;
 }
 
-static InitFunction DirtyDrivinFunc([]()
+D3DPRESENT_PARAMETERS* pPresentationParameters_RT9;
+uintptr_t d3dcall;
+void __stdcall D3D9CreateParamPatch()
```
Comment (Poliwrath) [inline OpenParrot/src/Functions/Games/Other/DirtyDrivin.cpp:166]: wrap this around a ``#if _M_IX86`` block

File: OpenParrot/src/Functions/WindowedDx9.cpp [hunk 2]
```diff
@@ -84,7 +84,7 @@ extern linb::ini config;
 
 static InitFunction initFunc([]()
 {
-	if (GameDetect::currentGame == GameID::BG4 || GameDetect::currentGame == GameID::BG4_Eng || GameDetect::currentGame == GameID::JLeague || GameDetect::currentGame == GameID::TER || GameDetect::currentGame == GameID::RumbleFish2)
+	if (GameDetect::currentGame == GameID::BG4 || GameDetect::currentGame == GameID::BG4_Eng || GameDetect::currentGame == GameID::JLeague || GameDetect::currentGame == GameID::TER || GameDetect::currentGame == GameID::RumbleFish2 || GameDetect::currentGame == GameID::DirtyDrivin))
```
Comment (Poliwrath) [inline OpenParrot/src/Functions/WindowedDx9.cpp:87]: remove the extra ) at the end of this

**Review by Poliwrath — CHANGES_REQUESTED** [review]
(from discord)

File: OpenParrot/src/Functions/Games/Other/DirtyDrivin.cpp (hunk 1 above)
Comment (00C0FFEE) [reply OpenParrot/src/Functions/Games/Other/DirtyDrivin.cpp:166]: ok done

**Review by Poliwrath — APPROVED** [review]

**Pull request merged**